redaxo/src/core/vendor/bin/var-dump-server

```diff
@@ -0,0 +1 @@
+../symfony/var-dumper/Resources/bin/var-dump-server
```

Comment:
Ist der nützlich für uns?

Reply:
Keine Ahnung, hab den selbst bisher noch nicht genutzt.
https://symfony.com/blog/new-in-symfony-4-1-vardumper-server

Soll ich alles bzgl var-dump-server erst mal wieder entfernen?

Reply:
Nein passt schon, ich dachte du hast vllt was im auge damit